README.md

```diff
@@ -95,6 +97,8 @@ To give you a sense of whether we will accept changes, you can use these heurist
 - Does an internet search for the fields show results in blogs/recommendations?
 - When was the deprecation (this can be hard to find) but was it at least 2 years ago if so?
 
+## Build Instructions
```

Comment:
this looks unfinished

Reply:
looks it was a merge error - this is a dupe from the top